project/FinaglePlugin.scala

```diff
@@ -20,7 +20,7 @@ object FinaglePlugin extends AutoPlugin {
    * because the newest release is not binary compatible with the older
    * versions it was checked against.
    */
-  val versions = Seq("22.7.0")
+  val versions = Seq("22.12.0", "22.7.0")
```

Comment:
Just to be explicit: I think we should remove `"22.7.0"` here, since we know `22.12.0` is not binary compatible with `22.7.0`. Once that's done, `rootFinagle/mimaReportBinaryIssues` should run cleanly, and we can add it to the build.